### DISCUSSION TITLE
Search through Different Relationship tables

### DISCUSSION THREAD

**Discussion by hasanakhtar** [Laravel-Backpack/community-forum#1057] [Q&A (Help)]
Hi,
I have a list view wherein column have 1-1 and 1-n and n-n relationships being displayed for a Model -Video. Like so:

        CRUD::addColumn([        
            'name'         => 'participant', // name of relationship method in the model
            'type'         => 'relationship',
            'label'        => 'Participants', // Table column heading
            'attribute' => 'name', // foreign key attribute that is shown to user
        ]);

       CRUD::addColumn([         
            'name'         => 'tag', // name of relationship method in the model
            'type'         => 'relationship',
            'label'        => 'Tags', // Table column heading   
            'attribute' => 'name', // foreign key attribute that is shown to user
        ]);

While the List operation lists the videos correctly, the search at the top right only searches through the db columns of "Video" table. I need to search to the other tables also (relationships)..ie say 'names' of tags in the tags table associated with the given video or the search by the names of participant in videos.

Have been toying with https://backpackforlaravel.com/docs/5.x/crud-columns#custom-search-logic but with no luck.

Request help. Thanks in advance !

**Comment by karandatwani92** [comment]
Hey @hasanakhtar 

Currently, the `relationship` column is not searchable by default. We are aware of it and it is in our current discussion to make `relationship` and other columns searchable & orderable by default.

For now
- To make 1-1 searchable, you can change to type to [`select`](https://backpackforlaravel.com/docs/6.x/crud-columns#select-1)
- For others, set[ Custom Search Logic for Columns](https://backpackforlaravel.com/docs/6.x/crud-columns#custom-search-logic-for-columns)

**Comment by jcastroa87** [comment]
Due to not activity, I will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.
